src/components/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import logo from '../images/logoMintWhite.png'
+
+const headerStyle = {
+  background: 'linear-gradient(180deg, #0E4B75 0%, #0D2145 100%)',
+  height: '212px',
+  fontSize: '64px',
+  fontWeight: 'bold' as const,
+  color: 'white',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexWrap: 'wrap' as const
+}
+
+const imageStyle = {
+  width: '300px',
+  height: 'auto',
+  marginLeft: '36px',
+  marginRight: '36px'
+}
+
+export function Header(): JSX.Element {
+  return (
+    <>
+      <div style={headerStyle}>
+        <div>Welcome to the</div>
```

Comment:
Avoid 'literal strings'.
Create a constant json file and import from there.

Reply:
Updated per comment. Created a `constant.json` file.